src/components/NewTabs/tabs.module.css

```diff
@@ -0,0 +1,27 @@
+.tabGroup p {
+    @apply text-sm;
+}
+
+.tabs{
+    composes: flex from global;
+}
+
+.tabs div {
+    @apply py-4 px-9 cursor-pointer text-center;
+    border: 1px solid rgb(var(--venia-global-color-border));
+    border-right: none;
+    border-bottom: none;
+}
+
+.tabs div:last-child {
+    border-right: 1px solid rgb(var(--venia-global-color-border));
```

Comment:
adjust to tailwind

Reply:
and check how it is possible to add tailwind styles for last-child https://v2.tailwindcss.com/docs/hover-focus-and-other-states#last-child

the same can be done for hover, focus, disabled and etc selectors